src/util/colors.js

```diff
@@ -3,4 +3,5 @@ export const colors = {
   lightGray: '#4d4d4d',
   white: '#fff',
```

Comment:
If in the future the app's color change you have to change each part where you put this colors, so I think is better to have names for colors like primary-color, secondary-color, acent-color, or something like that.